lib/Dialect/TritonGPU/Transforms/AccelerateAMDMatmul.cpp

```diff
@@ -63,13 +61,23 @@ warpsPerTileMFMA(tt::DotOp dotOp, const ArrayRef<int64_t> shape, int numWarps) {
   return ret;
 }
 
+SmallVector<unsigned, 2>
+warpsPerTileMFMA(tt::DotOp dotOp, const ArrayRef<int64_t> shape, int numWarps) {
```

Comment:
For mfma instructions, the output can also be 16x16 (mfma_16 instructions). Why assuming {32,32} here?

Reply:
Looks like 16x16 is not supported for mfma for now. Here is original code: https://github.com/ROCmSoftwarePlatform/triton/pull/423/files#diff-cbbb00df02bbf76f4e85d73e50cd2f27b8a1d8d8b4db306e26cebee62012118bL41

Should I add TODO comment?

Reply:
oops, looks like this is an error from existing code.
@joviliast just refactored this functions.

Seems we need to change this and measure performance for mfma 16x16